lib/std/std.zig

```diff
@@ -55,6 +59,7 @@ pub const compress = @import("compress.zig");
 pub const crypto = @import("crypto.zig");
 pub const cstr = @import("cstr.zig");
 pub const debug = @import("debug.zig");
+pub const debug_info = @import("debug_info.zig");
```

Comment:
This is bunch of loosely related utility functions. I am not sure that this belongs in the pubic interface. If so, it certainly needs a better name.